eraser-map.yaml

```diff
@@ -832,10 +831,10 @@ layers:
         data: { source: route_icon }
         draw:
             icons:
-                priority: 1
                 sprite: route-start
                 size: [38px,49px]
                 collide: false
+                anchor: top
```

Comment:
I tried `top` and `bottom` and `bottom` looked better to me, considering we have a bubble icon. 

But its a designer's choice, so I would leave the call for setting of anchors to you and Geraldine :D.